proto/minder/v1/minder.proto

```diff
@@ -342,6 +342,17 @@ service RepositoryService {
             relation: RELATION_REPO_CREATE
         };
     }
+    rpc ReconcileEntityRegistration(ReconcileEntityRegistrationRequest) returns (ReconcileEntityRegistrationResponse) {
+        option (google.api.http) = {
+            post: "/api/v1/provider/{provider}/register_all"
```

Comment:
I must have made a copy-paste in the original ticket. What I put in the additional bindings is what I think we should use as the REST API path, so `post: "/api/v1/provider/register_all"`. Which provider are we reconciling should be read from the context.

Reply:
I honestly thought this would be more appropriate as well. Thanks.